design/001-RPC-Interface-Specification.md

```diff
@@ -223,13 +228,12 @@ specified opening balance and challenge duration.
 *Parameters*
 
 * `Session ID`: [String] Unique ID of the session.
-* `PeerAlias`: [String] Alias of peer with whom channel should be opened.
```

Comment:
Since the `Alias` parameter has been removed, I wonder which peer the payment channel is opened with?

Reply:
The `Balances` field will already contain the `Alias`es of all all the participants to whom channel is to be opened with. Hence I felt, this field was redundant.

Reply:
I have updated the format of payment channel info to use lists instead of maps. This has two advantages:
1. We require one term less. Instead of using `Balances` for the map and `Balance` for the field in previous version, now there is just one term `Balances` for the list.
2. When we add support for multiple assets (like other ERC20 tokens) to be held in the same channel, this structure can be scaled without repeating the list of aliases for each asset.